iOS Client/HTTPClient/MBClient.swift

```diff
@@ -108,9 +108,9 @@ class MBClient: NSObject {
         }.resume()
     }
     
-    func getDevotionsWithCompletion(completion: @escaping (Data?, Error?) -> Void ) {
+    func getJSONFile(name: String, completion: @escaping (Data?, Error?) -> Void ) {
```

Comment:
so, this is also synchronous, but I'm OK with it exposing an async interface since we could conceivably change it to happen in the background. This hints to callers of this code that their completion block may not run on the main thread

Reply:
Yeah I'm going to keep it like this just in case in the future we have to get this file over the network